00-programming-fundamentals/installfest.md

```diff
@@ -31,50 +32,54 @@ If the Xcode Command-Line tools are not installed you can install it with:
 
 ## Homebrew
-[Homebrew](http://brew.sh/) is a **package manager** that simplifies installing software on macOS. Package managers are very common, and each language/framework/stack tends to have their own or preferred tool.
+
+[Homebrew](http://brew.sh/) is a **package manager** that simplifies installing software on macOS. Package managers are very common, and each language/framework/stack tends to have their own or preferred tool.  Homebrew should also have been installed in Jumpstart.
 
 To install, copypasta this into your terminal:
 
 `$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
 
 You may need to enter your computer password to complete the installation.
 
-### Trust but Verify
+### Trust but Verify Brew
+
 Run `$ brew doctor`. Brew is super great at telling you what else it may need. Sometimes there are file permissions to fix. Sometimes there's a warning that can be ignored. If there's anything ever wrong with Brew, `brew doctor` will usually be able to tell you what it is and how to fix it.
 
 ## RVM
+
 [RVM](http://rvm.io) (Ruby Version Manager) is a command-line tool which allows you to easily install, manage, and work with multiple ruby environments from interpreters to sets of gems. In essence, RVM is a package manager specifically for working with Ruby code.
 
 [**You should have installed RVM in JumpStart.**](https://github.com/Ada-Developers-Academy/jump-start/blob/master/lessons/07-environment-setup/notes/installfest.md)
 
-### Trust but Verify
+### Trust but Verify RVM
+
 - `$ rvm version` should tell you something like _rvm 1.26.11 (latest)..._
 
-## Upgrade and Use Ruby version 2.5.1
+## Upgrade and Use Ruby version 2.5.5
 
-We will use Ruby version 2.5.1 for the Ada curriculum _(last updated: early 2019)_.
+We will use Ruby version 2.5.5 for the Ada curriculum _(last updated: May 2019)_.
 
-- Use rvm to install the Ruby version 2.5.1 onto your computer: `$ rvm install 2.5.1`
+- Use rvm to install the Ruby version 2.5.5 onto your computer: `$ rvm install 2.5.5`
 - Verify that this version of Ruby was installed and that `rvm` recognizes it with `$ rvm list`
-- Use rvm to state that you want to use version 2.5.1 and you want to set it as the default version with `$ rvm use 2.5.1 --default`
+- Use rvm to state that you want to use version 2.5.5 and you want to set it as the default version with `$ rvm use 2.5.5 --default`
 
-**Note:** If you do not set this as the default version now, you will have to run `$ rvm use 2.5.1` every project! So go ahead and set it as default right now. ;)
+**Note:** If you do not set this as the default version now, you will have to run `$ rvm use 2.5.5` every project! So go ahead and set it as default right now. ;)
```

Comment:
"Now" implies that this is their one-and-only chance to set the default version.

```suggestion
**Note:** If you do not set this as the default version, you will have to run `$ rvm use 2.5.5` every project! So go ahead and set it as default right now. ;)
```